ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -849,6 +845,76 @@ function getPoint(
     return point;
 }
 
+/**
+ * @private
+ */
+function scatterProcessData(
+    series: Series
+): (boolean|undefined) {
+
+    if (series.isDirtyData) {
+        return;
+    }
+
+    if (!series.boost) {
+        series.data.length = 0;
+        series.isDirty = true;
+        return;
+    }
+
+    series.yAxis.setTickInterval();
```

Comment:
What's the purpose of this? It may potentially drag in a hairball of loops and stuff... If legit, we need a comment.

Reply:
It makes sure that I get proper extremes in the following lines.